docs/src/index.md

```diff
@@ -256,3 +256,17 @@ of a ```LinPred``` type use a default step factor of 1.  The value of
 ```linpred``` is the argument to the ```updatemu``` method for
 ```ModResp``` types.  The ```updatemu``` method returns the updated
 deviance.
+
+## Debugging failed fits
+In the rare cases when a fit of a generalized linear model fails, it can be useful
+to enable more output from the fitting steps. This can be done through
+the Julia logging mechanism by setting `ENV[JULIA_DEBUG] = GLM`. Enabling debug output
```

Comment:
```suggestion
the Julia logging mechanism by setting `ENV["JULIA_DEBUG"] = GLM`. Enabling debug output
```
I can't recall whether the RHS also needs quotes, nor whether this needs to be specified before Julia and/or the package is loaded.

Reply:
This version should be good.